rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc

```diff
@@ -1,7 +1,18 @@
 == Hardware Requirements
 
 * For optimal performance, use a dedicated computer for running the RPA Bot.
-
+* The following table shows the minimum and the recommended hardware requirements for running one process. To run more than one process in parallel, when using Secure Sessions, you need to add 2 GB of RAM plus the RAM needed to run the process for each parallel run to these recommended values.
++
+For example, let's say you want to run two different processes in parallel. The first process requires 1 GB RAM and should run in ten times in parallel. The second process requires 3 GB RAM and should run in five times in parallel. Then you need a computer with a minimum RAM of 63 GB:
```

Comment:
Can you streamline this a little? Maybe something like:

For example, to run two processes in parallel:

* Process 1 requires one GB of RAM and runs 10 times in parallel.
* Process 2 requires three GB of RAM and runs 5 times in parallel.

In this case, use the following formula to calculate that you need a computer with a minimum of 63 GB of RAM:

Reply:
The [style for numbers](https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-numbers) is to spell out 1-9 and use numerals for 10+